### DISCUSSION TITLE
About the copyright notice at the footer

### DISCUSSION THREAD

**Discussion by airinghost** [CaiJimmy/hugo-theme-stack#385] [Q&A] [answered]
Hi Jimmy, thanks for your open source of this new awesome Hugo theme.

I noticed that you mentioned in docs that the “Theme Stack designed by Jimmy” text and link should not be removed from the footer, so could I have your allowance to modify it, like “Build with Hugo & Jimmy’Stack” or something like that? The name “Stack” and Jimmy and their links would remain. 

Thanks again for your attention to my question.

**Comment by CaiJimmy** [comment, marked as answer]
Thanks for asking. It sounds good to me 👌.

**Reply by airinghost** [reply]
Thank you again.
